packages/api/portfolioDrafts/createPortfolioDraft.ts

```diff
@@ -38,7 +38,7 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
         Item: item,
       })
     );
-    return new ApiSuccessResponse<PortfolioDraftSummary>(item as PortfolioDraftSummary, SuccessStatusCode.CREATED);
+    return new ApiSuccessResponse<PortfolioDraftSummary>(item, SuccessStatusCode.CREATED);
```

Comment:
Does this not produce an error with Postman?
We are asserting most of our other calls with 
```item as <type>``` in the ApiSuccessResponse

Reply:
SonarCloud says that this is useless and I agree. We should either specify it in the `<>` _or_ with `as` _or_ neither. But not both. Not sure how Postman would have an issue. I'll provide a URL to test against internally in a moment.